### ISSUE TITLE
Log report.json to DC at the end of a run

### ISSUE THREAD

**Issue by pradh** [datacommonsorg/import#82]
Unlike the Import UI, which lets us monitor or debug usage, we have no insight into the stand-alone client.

One thing we could do is to ship the `report.json` content at the end of the run.  It should be well under 1MB in size.

Perhaps we can have a new recon API endpoint to save the `report.json` in a GCS bucket as a time-stamped file.

To make sure disconnected (or poorly-connected) users can use the tool, we should probably do this at the very end so even if interrupted, the command will have done its job.

Finally, if we also report back the GCS file path, users can then more conveniently share that URL while asking us questions (rather than sending along a report.json).

@chejennifer, @beets ... wdyt?

**Comment by beets** [comment]
Sounds good! Some additional stats might be helpful as well - like # of files / type, commands & flags used, etc.
Expect a lot of repeats though, much more so than with the current UI. Maybe a flag to turn off uploading the report.json to gcs (but still sends an analytics ping)?

**Comment by pradh** [comment]
Yeah, [command args](https://github.com/datacommonsorg/import/blob/master/util/src/main/proto/Debug.proto#L70) are part of the report, but there are probably some more things we can add to it.

Having a flag to turn off GCS upload SG.

**Comment by chejennifer** [comment]
Yeah, that sounds like a good idea!
